numpy/core/setup.py

```diff
@@ -34,6 +34,12 @@
 # useful to avoid improperly requiring SVML when cross compiling.
 NPY_DISABLE_SVML = (os.environ.get('NPY_DISABLE_SVML', "0") == "1")
 
+
+# Set NPY_DISABLE_HIGHWAY=1 in the environment to disable the Highway
+# library. This option only has significance on a AArch64 and is most
+# useful to avoid improperly requiring Highway when cross compiling.
```

Comment:
This seems copied from the SVML snippet, however the reason that is there for SVML is that that library gets shipped as object files rather than as source files. There shouldn't be an issue with Highway when cross-compiling I hope?